clerk-frontend/src/need-wizard/BSNStep.jsx

```diff
@@ -1,32 +1,21 @@
-import React, { Component } from 'react';
-
-class BSNStep extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      'bsn': ''
-    }
-
-    this.handleChange = this.handleChange.bind(this);
-  }
+import React from 'react';
 
+class BSNStep extends React.Component {
   handleChange(event) {
     if (this.props.bsnChanged) {
       this.props.bsnChanged(event.target.value)
     }
   }
 
-
-
   render() {
     return (
       <div className="bsn-form">
-        <form>
+        <form onSubmit={e => e.preventDefault()}>
           <div className="input">
             <label className="input__label" htmlFor="waardepapieren-bsn">
               Voer BSN in
             </label>
-            <input type="text" id="waardepapieren-bsn" className="input__control input__control--large input__control--text" onChange={this.handleChange} />
+            <input type="text" id="waardepapieren-bsn" className="input__control input__control--large input__control--text" onChange={e => this.handleChange(e)} />
```

Comment:
Is there a particular reason you used this lambda instead of binding the function in the constructor?

Reply:
No particular reason, I just removed the dead code above. I can revert it since using the binding function is inline with the codestyle of the other components

Reply:
I personally like that, because it's more explicit and prevents confusion about what "this" is.

Reply:
Since no state is used anyway I refactored the BSNSTep to a functional component and replaced the lambda with the `handleChange` function